src/Test/Perf/util/test_util.csx

```diff
@@ -255,28 +262,42 @@ void Log(string info)
     }
 }
 
-/// Logs the result of a finished process
+// Logs an error message.
+//
+// The actual implementation of this method may change depending on
+// if the script is being run standalone or through the test runner.
+void LogError(string info)
```

Comment:
I'm not sure I see the point of having Log and LogError.

Assuming that we'd want to see **all** of the logged statements regardless of if they are info or error, why differentiate them?

Maybe we should have Log do a flush on every WriteLine?